decimal_bench_test.go

```diff
@@ -121,6 +121,34 @@ func BenchmarkDecimal_RoundCash_Five(b *testing.B) {
 	}
 }
 
+func numDigits(b *testing.B, want int, val Decimal) {
+	b.Helper()
+	for i := 0; i < b.N; i++ {
+		if have := val.NumDigits(); have != want {
+			b.Fatalf("\nHave: %q\nWant: %q", have, want)
```

Comment:
Shouldn't we use the `%d` verb to print integers? :D

Reply:
Fixed, what I get for copy pasting code